lectures/lln_clt.md

```diff
@@ -552,10 +555,13 @@ We mentioned above that LLN can still hold sometimes when IID is violated.
 Let's investigate this claim further.
 
 Assume we have a AR(1) process as below:
+
 $$
 X_{t+1} = \alpha + \beta X_t + \sigma \epsilon _{t+1}
 $$
 
+and
+
 $$
 X_0 \sim \mathcal{N} \left(\frac{\alpha}{1-\beta}, \frac{\sigma^2}{1-\beta^2}\right)
```

Comment:
$N$ should be plain rather than caligraphic, as in https://python.quantecon.org/lln_clt.html